website/docs/tutorial/index.mdx

```diff
@@ -303,7 +303,7 @@ struct VideosListProps {
 }
 
 #[function_component(VideosList)]
-fn videos_list(VideosListProps { videos }: &VideosListProps) -> Html {
+fn VideosList(VideosListProps { videos }: &VideosListProps) -> Html {
```

Comment:
Thank you for the contribution.

However, I believe the original example is correct.
You cannot have a function component with its component name and function name to be the same.
You can try the updated example and it will produce a compile error.

Either the function needs to be assigned a snake case name like the original version, or you need to use a shorthand syntax.

A `inner` name is assigned to the function itself if the shorthand syntax is used.

```suggestion
#[function_component]
 fn VideosList(VideosListProps { videos }: &VideosListProps) -> Html {
```